app/api/tickets/rate/route.ts

```diff
@@ -14,12 +15,19 @@ export async function POST(req: Request) {
   const { ticketId, rate } = (await req.json()) as TAPITicketsRate
   const { error } = await supabaseAdmin.from("tickets").update({ is_open: false, rate: rate }).eq("id", ticketId)
   if (error) return NextResponse.json({ error: `Error in api/tickets/route.ts\n ${error.message}` }, { status: 400 })
+  // to don't show ticket it on support side
   await pusherServer.trigger("tickets", "tickets:close", {
     id: ticketId,
     is_open: false,
   } as ITicket)
+  // to clear messages on user side
+  await pusherServer.trigger(ticketId, "tickets:close", {
+    id: ticketId,
+    is_open: false,
+  } as ITicket)
   // TODO - after closing ticket don't show messages from currently closed ticket
   await getTicketId()
```

Comment:
`await getTicketId` may be not necessary as revalidatePath('/)

If it work without it - delete these lines